ts/smelter-node/src/manager/existingInstance.ts

```diff
@@ -62,4 +69,14 @@ class ExistingInstanceManager implements SmelterManager {
   }
 }
 
+function urlToString(url: URL): string {
+  let href = url.href;
+  // Removes trailing slash
+  if (href.slice(-1) === '/') {
```

Comment:
URL provides a constructor that allows building url from base url. I would use that instead of string operations